README.md

```diff
@@ -133,6 +133,19 @@ Now it's time to dig into the full etcd API and other guides.
 [security]: https://etcd.io/docs/latest/op-guide/security
 [tuning]: https://etcd.io/docs/latest/tuning
 
+## Test Status
```

Comment:
Would it be better to have it in CONTRIBUTING.md? or one of `Documentation/contributor-guide`

Reply:
My vote would be `CONTRIBUTING.md` as a new heading below the "Create a pull request" header: https://github.com/etcd-io/etcd/blob/main/CONTRIBUTING.md#create-a-pull-request

Reply:
Suggest to add one more doc something like `test-status.md` under `Documentation/contributor-guide`, and then add an item under [Find something to work on](https://github.com/etcd-io/etcd/blob/main/CONTRIBUTING.md#find-something-to-work-on) to point to the doc.

Either way works for me. The goal is to have people periodically check and drive to fix all the flaky test. Leave it to @jmhbnz to coordinate this via the bi-weekly bug triage meeting?